### PR TITLE
Checkout prebuilt dart into standalone dart location.

### PR DESCRIPTION
**Pull request by aam**
If we check out prebuilt dart executables into the same location where standalone dart sdk expects it, we don't need to customize that location for flutter vs dart. This simplifies scripts and fixes the problem where for andorid flutter build patched_sdk step doesn't use prebuilt dart binary, uses [slow] dart binary that targets arm, runs via simulator.

### CODE REVIEW

File: tools/dart/update.py [hunk 1]
```diff
@@ -28,54 +28,55 @@
 # Path constants. (All of these should be absolute paths.)
 THIS_DIR = os.path.abspath(os.path.dirname(__file__))
 MOJO_DIR = os.path.abspath(os.path.join(THIS_DIR, '..', '..'))
```
Comment (zanderso) [inline tools/dart/update.py]: Maybe rename MOJO_DIR -> FLUTTER_DIR while you're in here.

File: tools/dart/update.py [hunk 2]
```diff
@@ -73,8 +74,11 @@ def main():
     with zipfile.ZipFile(output_file, 'r') as zip_ref:
```
Comment (zanderso) [inline tools/dart/update.py]: Small cleanup suggestion: Maybe pull some or all of the zipfile extraction into a helper function.

File: tools/dart/update.py [hunk 3]
```diff
@@ -28,54 +28,55 @@
...
         get_sdk = True
 
   if get_sdk:
     # Completely remove all traces of the previous SDK.
-    if os.path.exists(DART_SDK_DIR):
-      shutil.rmtree(DART_SDK_DIR)
-    os.mkdir(DART_SDK_DIR)
+    if os.path.exists(dart_sdk_dir):
+      shutil.rmtree(dart_sdk_dir)
+    os.mkdir(dart_sdk_dir)
```
Comment (zanderso) [inline tools/dart/update.py]: I think this removal logic is no longer pointing at the right path. It should be removing dart_sdk_dir + 'dart-sdk'.

File: tools/dart/update.py (hunk 1 above)
Comment (aam) [inline tools/dart/update.py]: Done.

File: tools/dart/update.py (hunk 2 above)
Comment (aam) [inline tools/dart/update.py]: Done.

File: tools/dart/update.py (hunk 3 above)
Comment (aam) [inline tools/dart/update.py]: Done.

**Comment by aam** [comment]
Yes, the plan was to get https://codereview.chromium.org/2963053002/ at least lgtm'ed too before pinging flutter folks. Without that cl this won't work.

**Comment by aam** [comment]
Thanks!

**Pull request merged**